src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -76,8 +79,12 @@ object RuleUtils {
       val commonCnt = inputSourceFiles.count(entry.allSourceFileInfos.contains)
       val deletedCnt = entry.allSourceFileInfos.size - commonCnt
```

Comment:
can move this in the else block below, so it's only calculated when delete enabled